solar.rb

```diff
@@ -0,0 +1,160 @@
+# our customer will be able to interact with the program and
+# choose which pieces of information they can see at a time,
+# specifically by choosing which planet.
+# sudo gem install samsouder-titlecase --source=http://gems.github.com
+
+require 'rubygems'
+require 'titlecase'
+
+class SolarSystem
+
+  attr_accessor :planet_details, :choose_planet, :planets, :planet_list, :add_planet
```

Comment:
A couple of issues, 
1.  You are making these with `attr_accessor` which means a user could directly change the instance variables.  Instead I would recommend making them `attr_reader` unless you really **want** to give a user the ability to reassign the instance variables.  
2.  Why are `planet_details`, `choose_planet` `planet_list` and `add_planet` here?  They're **methods** not instance variables.